pkgs/development/tools/rust/cargo-nextest/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, rustPlatform, stdenv, libiconv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-nextest";
+  version = "0.9.10";
+
+  src = fetchFromGitHub {
+    owner = "nextest-rs";
+    repo = "nextest";
+    rev = "cargo-nextest-${version}";
+    sha256 = "0gbh990dd4514bfqh4x2nymh5w608ljp3s7akq100m4v723b6339";
+  };
+
+  cargoSha256 = "0c0i274dkz3jx9dzcxl9hyf3imiga8vb6m6jc5z1f1wdq3vknh9r";
+
+  cargoTestFlags = [ # TODO: investigate some more why these tests fail in nix
+    "--"
+    "--skip" "tests_integration::test_relocated_run"
+    "--skip" "tests_integration::test_run"
+    "--skip" "tests_integration::test_run_after_build"
+  ];
```

```suggestion
  checkFlags = [
     # TODO: investigate some more why these tests fail in nix
    "--skip::tests_integration::test_relocated_run"
    "--skip::tests_integration::test_run"
    "--skip::tests_integration::test_run_after_build"
  ];
```
normally I would expect it to look like this

---

My bad, it should be `--skip=test_name`.